test/adapter/json_api/belongs_to_test.rb

```diff
@@ -38,7 +38,11 @@ def test_includes_post_id
           end
 
           def test_includes_linked_post
-            @adapter = ActiveModel::Serializer::Adapter::JsonApi.new(@serializer, include: [:post])
+            actual = ActiveModel::SerializableResource.new(
```

Comment:
Is the helper globally available? I think it should be. If so, I'll update these tests.